test/integ/test_key_pair_authentication.py

```diff
@@ -248,8 +248,6 @@ def test_bad_private_key(db_parameters):
     )
 
     bad_private_key_test_cases = [
-        "abcd",
-        1234,
```

Comment:
I think removing the test is wrong. We should adapt the test to cover these cases, right?

Reply:
I don't think so – it's not an integration test concern I would say, and we've got the unit test to cover it. One has to keep in mind that these _are_ regression tests, not a complete certification. We're trying to avoid regressions.

Reply:
How about we move these two cases to the unit test as well, in addition to `class Bad`?

Reply:
Seems reasonable – good suggestion, I have pushed a change to the previous commit now in 818c3e8312777038681497e835f49541d14fef18, _moving_ those tests instead. Should help with continuity.